src/uwtools/cli.py

```diff
@@ -642,6 +643,15 @@ def _add_arg_keys(group: Group) -> None:
     )
 
 
+def _add_arg_leadtime(group: Group) -> None:
+    group.add_argument(
+        _switch(STR.leadtime),
+        help="Leadtime in hours",
+        required=True,
+        type=int,
+    )
```

Comment:
There will probably be a use case for sub-hourly output from our users (either immediately, or the not-to-distant future). How much trouble would it be to tackle that change here? I was originally thinking about it as a `timedelta` type object.

Reply:
I think a `--leadtime HH[:MM[:SS]]` option on the command line, immediately parsed to a `timedelta`, and a native `timedelta` in the API and deeper, is probably a good idea and straightforward to implement. Let me see if it won't overly complicate this PR.

Reply:
Implemented `--leadtime HH[:MM[:SS]]` support.